plugins/inputs/mesos/mesos.go

```diff
@@ -17,33 +17,57 @@ import (
 	jsonparser "github.com/influxdata/telegraf/plugins/parsers/json"
 )
 
+type Role string
```

Comment:
Metrics collected from master (the one gathered so far) differs dramatically from the one collected from slaves. I use this to distinguish sets of metrics which are being collected by this plugin. 
As you can see below there are some functions which operates using this values to determine in which context they are being invoked. 
I hope that answers your question.